conda/recipes/raft-dask/meta.yaml

```diff
@@ -26,13 +26,23 @@ requirements:
   build:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
-    - {{ compiler('cuda') }} {{ cuda_version }}
+    {% if cuda_major == "11" %}
+    - {{ compiler('cuda11') }} ={{ cuda_version }}
+    {% else %}
+    - {{ compiler('cuda') }}
+    {% endif %}
+    - cuda-version ={{ cuda_version }}
     - cmake {{ cmake_version }}
     - ninja
     - sysroot_{{ target_platform }} {{ sysroot_version }}
   host:
-    - cuda-python >=11.7.1,<12.0
-    - cudatoolkit ={{ cuda_version }}
+    {% if cuda_major == "11" %}
+    - cuda-python >=11.7.1,<12.0a0
+    - cudatoolkit
```

Comment:
As `compiler('cuda11')` already adds `cudatoolkit` to `host`, think we can drop this

```suggestion
```